dashboard/app/assets/stylesheets/application.scss

```diff
@@ -1239,6 +1239,7 @@ input.header_input {
 
 #codeApp {
   position: relative;
+  margin: env(safe-area-inset-top) env(safe-area-inset-right) env(safe-area-inset-bottom) env(safe-area-inset-left);
```

Comment:
Very interesting!

1. The "after" screenshot is a bit blurry, but it looks like the notch might overlap the content a bit still?
2. Is it worth adding a bit more padding so beyond the notch so it doesn't touch the UI?
3. Is there any way we could avoid the notch on the left while not losing space on the right?  Or are these designed to remain symmetrical?

Reply:
I think you are right about the minor overlap. I think it's a balance between 2 & 3. We don't know which way the user will rotate their phone, and it seems like the `safe-area` variable doesn't know which side the notch is on either. So we can add more padding, but it will add to both sides. And it will also add padding to every version of the lab, since I applied this to `#codeApp` generally.

Reply:
I don't love this solution because of the extra padding on the right, but it may be the best intermediate step. I'm hesitant to add more padding unless we can only add padding if the `safe-area-inset` is not `0`